spec/features/data_access/data_download_spec.rb

```diff
@@ -187,6 +189,34 @@
     end
   end
 
+  context "when a collection with more than 2000 granules has been selected for direct download" do
+    before :all do
+      load_page 'data/configure', project: 'C1000000561-NSIDC_ECS'
+      wait_for_xhr
+      choose 'Direct Download'
+      click_on 'Submit'
+      wait_for_xhr
+    end
+    context 'upon clicking a "View/Download Data Links" button' do
+      before :all do
+        click_link "View/Download Data Links"
+        wait_for_xhr
+      end
+
+      it "displays a page containing direct download hyperlinks for the collection's granules in a new window" do
+        within_last_window do
+          expect(page).to have_text("Collection granule links have been retrieved")
+          expect(page).to have_link("ftp://n5eil01u.ecs.nsidc.org/DP5/AMSA/AE_SI6.003/2011.08.03/AMSR_E_L3_SeaIce6km_V15_20110803.hdf")
```

Comment:
@jboukes This test is likely to fail because of the previously merged PR that removes the FTP links. But also, its only the 65th link in the list so it doesn't actually test that paging is working and more than 1999 links appear. 

I would update this test and instead of looking for a specific link I would ensure in some way that there are in fact more than 1999 link present.

Reply:
@abbottry You're right - I checked and live it is correctly doing 'https' links - the only reason this test was working is that the links were 'ftp' in the fixtures and the test was matching.

I deleted the references to this collection in the fixtures, and updated the test to include a count of links (3385).  I reran them to regenerate the granule fixtures file (so that the links are correctly https) and just pushed it all up.